src/ts/test/sort-by-rating-min.test.ts

```diff
@@ -0,0 +1,196 @@
+import { describe, expect, test } from "@jest/globals";
+import { sortByRatingMin } from "../modules/filter/compose/sort-by-rating-min";
+import { filterStore } from "../modules/filter/data/filter-store";
+
+describe("sort module", () => {
+  filterStore.sort = "ratingmin";
+  const send = [
```

Comment:
can you write like this
const data
const expectedData

Reply:
Discussed